Fluid/Values/FluidValue.cs

```diff
@@ -10,16 +10,13 @@ public abstract class FluidValue : IEquatable<FluidValue>
 #pragma warning restore CA1067
     {
         [Obsolete("WriteTo is obsolete, prefer the WriteToAsync method.")]
-        public abstract void WriteTo(TextWriter writer, TextEncoder encoder, CultureInfo cultureInfo);
-
-        public virtual ValueTask WriteToAsync(TextWriter writer, TextEncoder encoder, CultureInfo cultureInfo)
+        public virtual void WriteTo(TextWriter writer, TextEncoder encoder, CultureInfo cultureInfo)
         {
-#pragma warning disable CS0618 // Type or member is obsolete
-            WriteTo(writer, encoder, cultureInfo);
-#pragma warning restore CS0618 // Type or member is obsolete
-            return default;
+            WriteToAsync(writer, encoder, cultureInfo);
         }
 
+        public abstract ValueTask WriteToAsync(TextWriter writer, TextEncoder encoder, CultureInfo cultureInfo);
```

Comment:
Keep it virtual or this is a breaking change.

Reply:
I remembered you break many things in the past :)

Anyhow I will revert this